sites/skeleton.dev/src/content/docs/components/toast/react.mdx

```diff
@@ -0,0 +1,62 @@
+---
+layout: '@layouts/LayoutDoc.astro'
+---
+
+export const components = componentSet;
+
+import Example from '@examples/components/toast/Example.tsx';
+import ExampleRaw from '@examples/components/toast/Example.tsx?raw';
+import ExampleType from '@examples/components/toast/ExampleType.tsx';
+import ExampleTypeRaw from '@examples/components/toast/ExampleType.tsx?raw';
+import ExamplePlacement from '@examples/components/toast/ExamplePlacement.tsx';
+import ExamplePlacementRaw from '@examples/components/toast/ExamplePlacement.tsx?raw';
+import ExamplePromise from '@examples/components/toast/ExamplePromise.tsx';
+import ExamplePromiseRaw from '@examples/components/toast/ExamplePromise.tsx?raw';
+
+<Preview client:load>
+	<Fragment slot="preview">
+		<Example client:visible />
+	</Fragment>
+	<Fragment slot="code">
+		<Code code={ExampleRaw} lang="tsx" />
+	</Fragment>
+</Preview>
+
+## Type
+
+<Preview client:load>
+	<Fragment slot="preview">
+		<ExampleType client:visible />
+	</Fragment>
+	<Fragment slot="code">
+		<Code code={ExampleTypeRaw} lang="tsx" />
+	</Fragment>
+</Preview>
+
+Note: You can also use the more generic `create` method in combination with the `type` property when needing to create dynamic toast notifications.
```

Comment:
Let's make this the description under `## Type`

> You can use the generic `.create()` method in combination with a specific `type`, or opt for the shorthand methods if preferred: `.info()`, `.success()`, `.warning()`, or `.error()`.

This is for both React and Svelte docs.